images/scripts/lib/build-deps.sh

```diff
@@ -25,6 +25,13 @@ esac
 
 echo "$spec" | rpmspec -D "$OS_VER" -D 'version 0' --buildrequires --query /dev/stdin | sed 's/.*/"&"/' | tr '\n' ' '
 
+# We build anaconda rpms on fedora-37
+if [ "$OS_VER" = 'fedora 37' ]; then
+    $GET https://raw.githubusercontent.com/rhinstaller/anaconda/master/anaconda.spec.in | \
```

Comment:
We do have a script that installs all necessary Anaconda dependencies:

https://raw.githubusercontent.com/rhinstaller/anaconda/master/scripts/testing/install_dependencies.sh

Maybe we can extend the script so that it can also just return all the dependencies as a string ? Then it could be fetched & executed to get the dependencies. That way it could be more robust in the future as we do use and maintain that script.

Reply:
I'd prefer not to execute stuff, if possible.  Could you have the dependencies in a flatfile, instead?

Reply:
Indeed, that's a good point - still it would be good if the flat file would be somehow automatically maintained so that we don't have to update the deps manually. I will think about something. :)